charts/cloudnative-pg/values.yaml

```diff
@@ -149,6 +149,8 @@ monitoring:
     sidecarLabel: "grafana_dashboard"
     # -- Label value that ConfigMaps should have to be loaded as dashboards.
     sidecarLabelValue: ""
+    # -- Annotations that ConfigMaps should have to get Configured.
```

Comment:
What do you mean by "to get configured"?

Reply:
Sorry, I did a Mistake in the Description, but i don't know how to write the correct description .

I want to say:
# -- Annotations that ConfigMaps can have to get configured in grafana, for example:
## grafana_dashboard_folder: CNPG will Create a Folder CNPG in Grafanas dashboard overview and import the the Dashboard into this folder.